### DISCUSSION TITLE
How to detect did slide changed on click or on drag?

### DISCUSSION THREAD

**Discussion by pzubar** [nolimits4web/swiper#4579] [Q&A]
Hi!
Is there any way to detect the reason the slide changed? I would like to track how users interact with a slider so I need somehow to detect did the slide changed on click or on touch (swipe or drag on mobile).

Thank you in advance!

**Comment by nolimits4web** [comment]
Maybe this will help https://github.com/nolimits4web/swiper/issues/4346#issuecomment-801915187
